chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -162,6 +270,13 @@ class CurrentUser(
             callback: (Subscription) -> Unit
     ) = makeSingleCallback({ syncCurrentUser.subscribeToRoom(roomId, messageLimit, consumer) }, callback)
 
+    /**
+     * Subscribe to a [room] for multipart messages. You need to configure the [listeners] to listen
+     * for the events you're interested in, for more information on what events are possible see the [docs][https://pusher.com/docs/chatkit/reference/android#chat-events]
+     * You can configure a [messageLimit] which returns the 10 messages that were most recently sent to the room,
+     * each message will call the [onMultipartMessage] listener so you can determine how to display the message.
+     * Finally the [callback] will be called with a [Subscription] if the operation was successful.
```

Comment:
And the `Subscription` is there so that you can `cancel()` it when you no longer want to be receiving events.

Reply:
Ah. We aren't doing that in the demo app - should we?

I've written it to read: `You will need to use this subscription to `cancel()` when you no longer want to receive any new events in this room.`